input-profile.js

```diff
@@ -0,0 +1,176 @@
+import {Component, Property} from '@wonderlandengine/api';
+import {vec3, quat} from 'gl-matrix';
+const _TempVec = vec3.create();
```

Comment:
We write those in camelCase. I don't mind the trailing `_`, we can keep that. We just need to be consistent with the rest of the codebase